### ISSUE TITLE
How to test the network?

### ISSUE THREAD

**Issue by bareblackfoot** [thtrieu/darkflow#676]
I want to test the network with the trained model. How can I test the model with the Pascal VOC 2007 test set?
Additionally, How many times should I train the network? the number of epochs is designated as 2000, but I think it is too many (compared to the Faster R-CNN). Should I use the pre-defined parameters?

**Comment by melonetern** [comment]
yes, you should use the default configuration and only follow the readme.md.

**Comment by bareblackfoot** [comment]
@melonetern I want to see the mAP result. I only found that the network test with demo images.

**Comment by andikira** [comment]
i have same issues, how to measure mAP from prediction json file ? 
i've seen some issues in this repo but i can't find the solution.

**Comment by andikira** [comment]
@bareblackfoot you have to see this repo https://github.com/Cartucho/mAP in this issue https://github.com/Cartucho/mAP/issues/3 .This repo work perfectly with darkflow format annotation and prediction file.

**Comment by bareblackfoot** [comment]
@andikira  Thank you! I got the mAP result.

**Comment by lyq998** [comment]
@bareblackfoot Hi, I am trying to measure the mAP of yolov1 on VOC 2007. I want to get the same mAP reported in the paper [https://arxiv.org/pdf/1506.02640.pdf](https://arxiv.org/pdf/1506.02640.pdf) which is 63.4%. But I only got mAP nearly 13%, and the confidence of each test image is relatively low (such as 0.02, 0.03). Very few images were tested with more than 50% confidence.

So I wonder if you got the same result as the original paper? Or is there something wrong with my operation?

Here are the details of my command:
`./flow --model cfg/v1.1/yolov1.cfg --load bin/yolov1.weights --imgdir test_dataset/VOCdevkit/VOC2007/JPEGImages --json --threshold 0.01`
where yolov1.weights is download from [http://pjreddie.com/media/files/yolov1/yolov1.weights](http://pjreddie.com/media/files/yolov1/yolov1.weights)
And I use this repo [https://github.com/Cartucho/mAP](https://github.com/Cartucho/mAP) to test mAP.

Furthermore, I cannot run this code on GPU, because it always gives an error 'out of memory'. I tried to use 2080 Ti (11 GB) and Tesla V100 (16 GB). However, it always gives this error, even with batch size=1.
